src/Row.js

```diff
@@ -170,7 +170,7 @@ export default class Row extends Component {
   _relocate(nextLocation, animated) {
     this._location = nextLocation;
 
-    if (animated) {
+    if (animated && !this._isAnimationRunning) {
```

Comment:
this is not correct, there is possible case, when location set programmatically, before previous relocation animation not ended. Also I doubt, that this change hugely improve performance